pipeline/matcher/scripts/get_graph.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import boto3
+import click
+from graphviz import Digraph
+
+from dynamo import get_graph_table_row, get_graph_component
+from elastic import get_pipeline_storage_es_client, get_nodes_properties
+
+source_type_labels = {
+    "sierra-system-number": "Sierra",
+    "miro-image-number": "Miro",
+    "mets": "METS",
+    "calm-record-id": "CALM",
+}
+
+
+@click.command()
+@click.argument("index_date")
+@click.argument("work_id")
+def main(index_date, work_id):
+    session = boto3.Session(profile_name="platform-read_only")
+    # Necessary for reading secrets
+    dev_session = boto3.Session(profile_name="platform-developer")
```

Comment:
Minor: I would prefer these use the full role ARNs; we have helpers for getting these. It's a bit more work, but it means you don't need `~/.aws/credentials` set up.

Separately: I wonder if we should have a `-secrets` role, that gives you read_only plus secrets access?